m4/ax_python_module.m4

```diff
@@ -21,13 +23,13 @@
 #   and this notice are preserved. This file is offered as-is, without any
 #   warranty.
 
-#serial 6
+#serial 7
 
 AU_ALIAS([AC_PYTHON_MODULE], [AX_PYTHON_MODULE])
 AC_DEFUN([AX_PYTHON_MODULE],[
     if test -z $PYTHON;
     then
-        PYTHON="python"
+        PYTHON="$3"
```

Comment:
Doh; I didn't mean to mark the parameter optional, but it seems like a good idea to me. Now the third parameter defaults to "python3," but I'm definitely open to other suggestions.